vllm/sequence.py

```diff
@@ -370,6 +383,21 @@ def __eq__(self, other: object) -> bool:
                 and self.logprobs == other.logprobs)
 
 
+class SequenceGroupOutputs:
+    """The model outputs associated with a sequence group."""
+
+    def __init__(
+            self, samples: List[SequenceOutputs],
+            prompt_logprobs: Optional[List[Optional[Dict[int,
+                                                         float]]]]) -> None:
```

Comment:
style nit: This looks nicer to me.
```suggestion
    def __init__(
        self,
        samples: List[SequenceOutputs],
        prompt_logprobs: Optional[List[Optional[Dict[int, float]]]],
    ) -> None:
```

Reply:
Fixed!